scene/2d/tile_map.cpp

```diff
@@ -459,10 +459,12 @@ void TileMap::_update_dirty_quadrants() {
 					Transform2D xform;
 					xform.set_origin(offset.floor());
 
-					Vector2 shape_ofs = tile_set->tile_get_shape_transform(c.id, 0).get_origin();
+					Vector2 shape_ofs = tile_set->tile_get_shape_offset(c.id);
 
 					_fix_cell_transform(xform, c, shape_ofs + center_ofs, s);
 
+					//xform *= shapes[i].shape_transform;
```

Comment:
Please remove commented out code. If there's a reason to keep it, there must be a comment explaining why.